ts-framework/functions/src/build/mcp.ts

```diff
@@ -60,6 +60,14 @@ async function buildFunctionsManifest(options: {
     name: string;
     description?: string | undefined;
     inputSchema: unknown;
+    _meta?: unknown;
+  }>;
+  resources?: Array<{
+    name: string;
+    description?: string | undefined;
```

Comment:
I think something we need to figure out. Although descriptions seemingly are optional at the MCP SDK level for tools and resources our Gram Functions system requires them. Not totally sure what to do about this